### ISSUE TITLE
Unfriendly warning when terminals cannot be split

### ISSUE THREAD

**Issue by jrieken** [Microsoft/vscode#59309]
re #59266

* Keep pressing the split button
* At one point the warning shows

I think we can do better than that... Why don't we disable the action instead of showing the warning? Also, why is this a warning and not an information message?

<img width="618" alt="screen shot 2018-09-25 at 11 10 18" src="https://user-images.githubusercontent.com/1794099/46004649-abdf0980-c0b3-11e8-980b-daf6ab8728be.png">

**Comment by bpasero** [comment]
Yeah also found this a bit weird. Fyi we do not prevent splitting of editors in the grid widget and have not heard complaints so far. So I am not sure why we need it here.

**Comment by alexr00** [comment]
We had one complaint about infinite splitting which is why I added this. I agree that behavior should generally be consistent between editor and terminal for splitting but at the same time allowing the user to put their terminal into a broken state isn't great either. 

I was concerned that simply disabling the split operation wouldn't make why it's disabled clear, but it is more intrusive than just disabling it!

This issue has already gotten more thumbs ups than the original (#45678), so clearly something needs to be done 😄 @Tyriar, let me know what you think!

**Comment by bpasero** [comment]
I think disabling the split action when we think it should be prevented is the best solution.

**Comment by Tyriar** [comment]
@alexr00 I guess we could watch for when the panel resizes and disable/enable if there's room.

**Comment by alexr00** [comment]
I've created PR for changing the warn to info. The disable/enable for split is more complicated so it would be better to do during the next milestone. Issue #59414 tracks that!
